R/FilterStates.R

```diff
@@ -477,13 +479,6 @@ FilterStates <- R6::R6Class( # nolint
                   teal_slice(dataname = private$dataname, varname = input$var_to_add)
                 )
               )
-              logger::log_debug(
-                sprintf(
-                  "FilterStates$srv_add@2 added FilterState of variable %s, dataname: %s",
-                  input$var_to_add,
-                  private$dataname
-                )
-              )
             }
           )
 
```

Comment:
```suggestion

          # Extra observer to destroy that clears all input values
          private$observers$input_remove <- list(
            destroy = function() {
              if (!session$isEnded()) {
                lapply(session$ns(names(input)), .subset2(input, "impl")$.values$remove)
              }
            }
          )
```

Reply:
I do like this suggestion. I'll implement this instead. Tested locally, works as expected 👍

Reply:
Thanks @averissimo for this suggestion. I've included remove-inputs-callback to the list of `observers`. I've changed `observers` to `shiny_bindings` instead as it is not observers only. Feel free to suggest different field name (`shiny_bindings` is not precise enough as it contains observers and `list(destroy = function() kill input)`